src/main/java/duke/io/Ui.java

```diff
@@ -0,0 +1,171 @@
+
+    /**
+     * Returns a formatted String that signifies that the user inputs the date and time in a wrong format.
+     *
+     * @return String that shows a date time format error.
+     */
+    public String showDateTimeException() {
+        return format("Date'T'time inputted is not of valid format: YYYY-MM-DDThh:mm");
+    }
+
+    /**
+     * Returns a formatted String that shows the outcome of a the command the user inputted.
+     * Used for commands involving a single word or for addition of tasks.
+     *
+     * @param c Type of command the user has inputted.
+     * @param list Current TaskList used.
+     * @return String that shows the outcome of a the command the user inputted.
+     */
+    public String displayCommand(Command.Commands c, TaskList list) {
+        switch (c) {
+        case HELP:
+            return format("list : Returns all tasks added\n",
+                    "todo <description> : Adds a todo task\n",
+                    "find <description> : Returns all tasks with <description> in their description\n",
+                    "event <description> /at <time: YYYY-MM-DDThh:mm> : Adds an event at time <time>\n",
+                    "deadline <description> /by <time: YYYY-MM-DDThh:mm> : Adds a task with deadline at time <time>\n",
+                    "done <index> : Marks the task at <index> as done\n",
+                    "delete <index> : deletes the task at <index>\n",
+                    "at <time: YYYY-MM-DDThh:mm> : Returns all events up till <time>\n",
+                    "by <time: YYYY-MM-DDThh:mm> : Returns all tasks with deadline due before or at <time>\n",
+                    "all <time: YYYY-MM-DDThh:mm> : Returns all timed tasks with times up till <time>");
+        case LIST:
+            return format(list.returnItems());
+        case TODO:
+            //Fallthrough
+        case EVENT:
+            //Fallthrough
+        case DEADLINE:
+            return format("Got it. I've added this task:",
+                    "  " + list.returnLastTask(), list.returnItemCount(0));
+        default:
+            return "Oops an error has occurred";
+        }
+    }
+
+    /**
+     * Returns a formatted String that that shows the outcome of a the command the user inputted.
+     * Used for commands involving involving retrieval of tasks based on date and time.
+     *
+     * @param c Type of command the user has inputted.
+     * @param list Current TaskList used.
+     * @param dt Date and Time used by the command.
+     * @return String that shows the outcome of a the command the user inputted.
+     */
+    public String displayCommand(Command.Commands c, TaskList list, LocalDateTime dt) {
+        switch (c) {
+        case AT:
+            return format(list.getEventsAt(dt));
+        case BY:
+            return format(list.getDeadlinesBy(dt));
+        case ALL:
+            return format(list.getAllBy(dt));
+        default:
+            return "Oops an error has occurred";
+        }
```

Comment:
Sorry for dupulicates, but how about reconsidering the coding standard for switch-case?

Reply:
Same as above. It's correct!